lib/OpenQA/WebAPI/Plugin/RateLimits.pm

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::WebAPI::Plugin::RateLimits;
+use Mojo::Base 'Mojolicious::Plugin', -signatures;
+
+
+use Time::Seconds;
+
+sub register {
+    my ($self, $app) = @_;
```

Comment:
Note: I actually tried to use signatures here. But it gave me errors from plugin loading code. I guess I would need to fix this on another layer?

Reply:
https://docs.mojolicious.org/Mojolicious/Plugin
```
sub register ($self, $app, $conf) {
```
:)

Reply:
Oooooohhhhhhh 🤦🏾‍♀️